engine/src/main/java/org/terasology/engine/module/rendering/package-info.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A Directed Acyclic Graph (DAG) is the type of data structure that is at the core of the Renderer.
```

Comment:
Would it make sense to link other classes and interaces here? See https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDJGIJB